index.html

```diff
@@ -184,6 +184,50 @@ <h2 class="about-title">About Me</h2>
             <li class="skill">Gitlab</li>
         </ul>
     </section>
+    <footer>
+        <section>
+            <h2 class="titleFooter">
+                I'm always interested in hearing about 
+                new projects, so if you'd like to chat 
+                please get in touch.
+            </h2>
+            <form action="" class="form">
+                <input class="form-list" type="text" placeholder="Full Name" required maxlength="30">
+                <input class="form-list" type="email" placeholder="Email Address" required>
+                <textarea class="form-textarea" placeholder="Write me something..." maxlength="500" required></textarea>    
+                <button class="botn">Get in touch</button>
```

Comment:
- [ ] Kindly make sure to implement the button interactions required in this project.
  
  ![image](https://github.com/AndresEspi/portafolio/assets/102612396/66cbd785-d812-48f6-a2e9-34207a0bb426)